agent/core/src/main/java/org/jolokia/config/ConfigKey.java

```diff
@@ -197,6 +197,16 @@
      */
     DISCOVERY_AGENT_URL("discoveryAgentUrl",true,false),
 
+    /**
+     * IPv4 Address for Jolokia's Multicast group.
+     */
+    MULTICAST_GROUP("multicastGroup",true,false),
```

Comment:
```suggestion
    MULTICAST_GROUP("multicastGroup",true,false, "239.192.48.84"),
```

You can directly add a default value to the overloaded enum constructor with 4 args.